src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
```

Comment:
```suggestion
public class ColorSupplier {
```
Тут наслідування також не потрібно. Воно доцільне в тих випадках, коли в тебе частина полей повторюється в схожих класах, наприклид різні види автомобілей SportCar, Truck, Bus мають спільну характеристику (поле) - потужність. Тоді щоб не писати це поле в 3 класах, ти його перевикористовуєш і виносиш в спільний батьківський клас Machine. І тоді буде, що Truck extends Machine і він автоматично унаслідує це поле від Machine  і наслідування буде доцільно бо допомогло уникнути дублювання коду